corehq/apps/cloudcare/static/cloudcare/js/form_entry/web_form_session.js

```diff
@@ -121,11 +121,16 @@ hqDefine("cloudcare/js/form_entry/web_form_session", function () {
             requestParams['tz_offset_millis'] = (new Date()).getTimezoneOffset() * 60 * 1000 * -1;
             requestParams['tz_from_browser'] = Intl.DateTimeFormat().resolvedOptions().timeZone;
 
+            var newData = new FormData();
+            _.each(requestParams, function (value, key) {
+                newData.append(key, value);
+            });
             return $.ajax({
                 type: 'POST',
                 url: self.urls.xform + "/" + requestParams.action,
-                data: JSON.stringify(requestParams),
-                contentType: "application/json",
+                data: newData,
+                contentType: false,     // or "multipart"? right now formplayer is 403ing either way, it wants json
```

Comment:
I think this would have to be dynamic based on the action. 'answer' requires `applicaiton/json` whereas 'answer_media' requires `multipart/form-data`.

I tested this locally and I think the issue is actually that this request is being converted to a multipart request in which ALL the data is being encoded as multipart data and not as JSON data. Even normal question answers are being encoded this way.

I think you're going to need to do something like this:

```javascript
let newData = null,
    contentType = "application/json";
if (requestParams.action === 'answer_media') {
    newData = new FormData();
    newData.append("answer", JSON.stringify(requestParams));
    newData.append("file", requestParams.file);
    contentType = "multipart/form-data";
} else {
    newData = JSON.stringify(requestParams);
}

Reply:
Thank you, updated in efb96aa66069677e23f3976842889cace74b7fdc, I was confused about how to include both json and form data in the same request.